core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -653,6 +653,19 @@ class ZkUtils(val zkClient: ZkClient,
     (dataStr, stat)
   }
 
+  def readData(path: String, returnNoneIfPathNotExists: Boolean): Option[String] = {
```

Comment:
It's just a readability issue, `readDataMaybeNull(....)._1` is opaque compared to `readData(...)`. And if we don't need `Stat`, it seems pretty pointless.